src/index.js

```diff
@@ -17,11 +18,8 @@ export default function App () {
       </BrowserRouter>
 
     <Services />
-
+    <Footer />
```

Comment:
issue: This code will place the Footer above the service map area, which I don't think is how the design looks.

Reply:
Where I suppose to display the footer?